src/System.Runtime/tests/System/TimeZoneInfo.cs

```diff
@@ -159,15 +160,20 @@ public static void NearMinMaxDateTimeOffsetConvertTest()
         VerifyConvert(new DateTimeOffset(DateTime.MinValue.AddHours(5), new TimeSpan(-3, 0, 0)), s_strPacific, new DateTimeOffset(DateTime.MinValue, new TimeSpan(-8, 0, 0)));
 
         VerifyConvert(DateTime.MaxValue, s_strPacific, s_strSydney, DateTime.MaxValue);
-        VerifyConvert(DateTime.MaxValue.AddHours(-19), s_strPacific, s_strSydney, DateTime.MaxValue);
+        if (Interop.IsWindows) // [ActiveIssue(846, PlatformID.AnyUnix)]
```

Comment:
Ok, thanks.  At this point, we might want a separate issue specific to cleaning up everything remaining in time zone support on Unix, and tag each remaining thing in the code with that new number.  That'll make it easier to know when we're "done" with it.  The 846 grab-bag of globalization issues is pretty big.